typescript/cli/package.json

```diff
@@ -66,6 +66,9 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "type": "module",
+  "exports": {
+    ".": "./dist/src/index.js"
```

Comment:
this should match `main` and `types`. So the output path `/dist/src/index.js` or `/dist/index.js`? The later is slightly preferable but nbd either way as long as the fields are consistent

Reply:
update to match